packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -1,20 +1,18 @@
-import React from 'react'
-import { ComponentMeta } from '@storybook/react'
+import { Meta } from '@storybook/react'
 
 import { Button } from './Button'
 import { Box } from '../Box/Box'
-import { Variants } from './styles.css'
 import { IconSolidCheveronDown, IconSolidSave } from '../icons'
 
 export default {
 	title: 'Components/Button',
 	component: Button,
-} as ComponentMeta<typeof Button>
+} as Meta<typeof Button>
 
 export const ButtonVariants = () => {
-	const kind: Variants['kind'][] = ['primary', 'secondary']
-	const emphasis: Variants['emphasis'][] = ['high', 'medium', 'low']
-	const size: Variants['size'][] = ['xSmall', 'small', 'medium']
+	const kind: any[] = ['primary', 'secondary', 'asdf']
+	const emphasis: any[] = ['high', 'medium', 'low']
+	const size: any[] = ['medium', 'small', 'xSmall']
```

Comment:
I couldn't get these types to work anymore. Didn't seem worth spending too much time on, and felt like it's something that should be fixed in VE at some point.